exercises/concept/annalyns-infiltration/.docs/instructions.md

```diff
@@ -6,9 +6,9 @@ After some time spent following her best friend's trail, she finds the camp in w
 
 Having found the kidnappers, Annalyn considers which of the following actions she can engage in:
 
-- Fast attack: a fast attack can be made if the knight is sleeping, as it takes time for him to get his armor on, so he will be vulnerable.
+- Fast attack: a fast attack can be made if the knight is sleeping, as it takes time for him to get his armour on, so he will be vulnerable.
```

Comment:
`armour` is the [British spelling](https://grammarist.com/spelling/armor-armour/). 

Exercism uses American spelling for the content (see the [Exercism style guide](https://exercism.org/docs/building/markdown/style-guide)).